.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,73 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone the go-tarantool connector
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/go-tarantool
+
+      - name: Download the tarantool build artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: Install tarantool
+        # Now we're lucky: all dependencies are already installed. Check package
+        # dependencies when migrating to other OS version.
+        run: sudo dpkg -i tarantool*.deb
+
+      - name: Get the tarantool version
+        run: |
+          TNT_VERSION=$(tarantool --version | grep -e '^Tarantool')
+          echo "TNT_VERSION=$TNT_VERSION" >> $GITHUB_ENV
+
+      - name: Setup golang for connector and tests
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.13
+
+      - name: Run base tests
+        run: |
+          mkdir snap xlog
+          TNT_PID=$(tarantool ./config.lua > tarantool.log 2>&1 & echo $!)
+          go clean -testcache && go test -v
+          kill $TNT_PID
+
+#      - name: Run queue tests
```

Comment:
Nit: it would be good to mention an issue, after which we expected to uncomment those lines.

Reply:
Fixed, thanks.